src/pages/Landing/index.tsx

```diff
@@ -52,104 +52,111 @@ export default () => {
   };
 
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
+  export const fetchCustomers = async () => {
+    const response = await axios.get('src/api/CustomerApi');
+    return response.data;
+  };
```

Comment:
You don't need to implement an api call, there are functions there that will simulate that for you:  https://github.com/content-services/React-typescript-demo/blob/main/src/api/CustomerApi.ts#L20-L30

you only need to call these function, not try to call out to an actual (real) api